after-your-talk.txt

```diff
@@ -52,3 +52,7 @@ During Q&A you may have someone point out that you said something inaccurate,
 or missed pointing out something useful. Accept this sort of feedback happily.
 If you made a mistake, admit to it immediately. Being totally honest will win
 you more credibility than pretending to be right all the time.
+
+4. Admit what you don't know.
+
+If you're not sure, say so. Others seeing that you knowing that you are wrong is better than it seeming like you do not know you are wrong. It's better than having someone disprove you. In this case, say that you will get back to them, get some sort of contact information, and compliment their question. After admitting your uncertainty, you are certainly welcome to make a guess, and provide your rationale for your thinking.
```

Comment:
@elizabethsiegle This is (really) a minor point, but it looks like the rest of these text documents are hard-wrapped at 80 columns. In the Vi editor, you would do:

    <Esc> :set tw=80
    
then move the cursor over the paragraph in question and press `gqq`. The Unix command [`fold`](http://man.cx/fold) may work, too, if you're not familiar with Vi.

If you use Windows, you might just need to enable the "margin guide" in the settings of your editor and then manually insert carriage returns. In Notepad++, it's under _Preferences_ &rarr; _Editing_ &rarr; _Vertical Edge Settings_.